knimin/lib/tests/test_configuration.py

```diff
@@ -22,6 +22,14 @@ def test_init(self):
         with self.assertRaises(IOError):
             KniminConfig('does not exist')
 
+        # test that expection is raised if not all sections are specified
+        self.config = tempfile.NamedTemporaryFile()
```

Comment:
I checked. You are right. The tests also works without self. I changed the according code block as you suggested. Thanks for enlighten me here!